packages/api/src/add-build.js

```diff
@@ -92,22 +101,67 @@ async function addTestRun (testCase, buildInfo, dbRepo) {
   );
 }
 
-async function updateTest (testCase, buildInfo, dbRepo) {
-  // first read the test
-  const prevTest = await dbRepo.collection('tests').doc(testCase.encodedName).get();
+async function updateQueue (isFirstBuild, testCase, buildInfo, dbRepo) {
```

Comment:
Here is the new function that ultimately determines which tests we will be updating

Reply:
@bcoe